app/mailers/effective/polls_mailer.rb

```diff
@@ -70,7 +70,6 @@ def effective_email_templates_assigns(poll_notification, user)
       raise('expected poll to be persisted') unless poll&.persisted?
 
       {
-        subject: poll_notification.subject,
```

Comment:
This was the hacky way we passed the subject in with the old version of the code. Now we do it explicitly.